org.opent2t.sample.binaryswitch.superpopular/com.smartthings.binaryswitch/js/package.json

```diff
@@ -24,6 +25,8 @@
   "devDependencies": {
     "optimist": "*",
     "ava": "^0.15.2",
-    "es6-sleep": "*"
+    "es6-sleep": "*",
+    "opent2t-device-smartthingshub": "file:../../../org.opent2t.sample.hub.superpopular/com.smartthings.hub/test",
+    "opent2t-device-binaryswitch": "file:../../test"
```

Comment:
Yes, it's getting the mock hub and the mock binary switch.  the "[file:...](file:...)" syntax tells npm to look for the package locally rather than trying to obtain it from a server.  This allows us to package the mock devices and require them like any other node package without having to publish them.

---
In reply to: [90383696](https://github.com/openT2T/translators/pull/135#discussion_r90383696) [](ancestors = 90383696)